browser/notifications/BUILD.gn

```diff
@@ -1,31 +1,18 @@
 source_set("notifications") {
   # Remove when https://github.com/brave/brave-browser/issues/10657 is resolved
   check_includes = false
```

Comment:
As we have refactored this area of code, can you please remove `check_includes = false` and make sure `gn_check` is passing